cloud_dashboard/src/molecules/DataTableRow.tsx

```diff
@@ -2,6 +2,29 @@ import React from 'react';
 import DataColumn from 'model/DataColumn';
 import DataRecord from 'model/DataRecord';
 
+/**
+ * TD tag of DataTable.
+ *
+ * @param text Message.
+ */
+const DataTableData = ({text}: {
+  text: string | number
+}) => {
+  if (typeof text === 'string' && text.includes('\n')) {
+    const temp = text.split('\n');
```

Comment:
This can be:

```suggestion
    const data = text.split('\n');
```

Or more meaningful variable name.